pythonStacker/plotHistogramsRedo.py

```diff
@@ -240,12 +256,16 @@ def get_lumi(years):
         "2016": 36.3,
         "2017": 41.5,
         "2018": 59.8,
+        # "Run2": 167.0,
     }
+    sum_years = {"2016", "2017", "2018"}
     for year in years:
-        total_lumi += lumi[year]
+        if year in sum_years:
+            total_lumi += lumi[year]
```

Comment:
Why are these lines added?

Reply:
To fix the value of the Run 2 cross-section (was summing 2016, 2016preVFP, and 2016postVFP), it is indeed contrite but I needed a quick solution

Reply:
ok but that means the 3 2016's are loaded so that you're plotting the wrong things. So this shouldn't be done, instead you need to make sure you just specify -y 2016 2017 2018 or with 2016 split. The lumi being wrong is just the signal to check and modify this